content/docs/introducing-jsx.md

```diff
@@ -87,37 +96,38 @@ function getGreeting(user) {
 }
 ```
 
-### Specifying Attributes with JSX {#specifying-attributes-with-jsx}
+### Attributen specificeren met JSX {#specifying-attributes-with-jsx}
 
-You may use quotes to specify string literals as attributes:
+Je kunt aanhalingstekens gebruiken om string literals als attributen te specificeren:
 
 ```js
 const element = <div tabIndex="0"></div>;
 ```
 
-You may also use curly braces to embed a JavaScript expression in an attribute:
+Je kunt ook accolades gebruiken om een JavaScript expressie in een attribuut te plaatsen:
 
 ```js
 const element = <img src={user.avatarUrl}></img>;
 ```
 
-Don't put quotes around curly braces when embedding a JavaScript expression in an attribute. You should either use quotes (for string values) or curly braces (for expressions), but not both in the same attribute.
+Zet geen aanhalingstekens om accolades als je een JavaScript expressie wilt gebruiken in een attribuut. 
+Je gebruikt of aanhalingstekens (voor string waarden) of accolades (voor expressies), maar niet beide in hetzelfde attribuut.
```

Comment:
```suggestion
Je gebruikt ofwel aanhalingstekens (voor string waarden) ofwel accolades (voor expressies), maar niet beide in hetzelfde attribuut.
```
Seems clearer to me, but might very well be a Dutch-Flemish thing again.

Reply:
That sounds Flemish to me :).